pig/src/main/java/org/jboss/pnc/bacon/pig/impl/utils/FileDownloadUtils.java

```diff
@@ -99,6 +99,14 @@ public static void setAttempts(int attempts) {
             attempts = MAX_ATTEMPTS;
         }
 
+        if (attempts <= 0) {
+            log.warn("Number of download attempts has to be a positive integer. Setting to {}", 1);
```

Comment:
The log message should use `, DEFAULT_ATTEMPTS` not `1`, in case you decide change it later.

Reply:
:+1: